package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-connector-cassandra",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
there is no need to change the version number here.  When this module gets published, our tool will increment the version number accordingly.

Reply:
@dhmlau done. Reset the change. Please review again. Thanks.